### PR TITLE
Fix 169 token support

### PR DESCRIPTION
**Pull request by CFDevelopment**
- closed issue 169
- removed old token list & added newly proposed tokens
- enable tokens page removed & Ethereum set as default upon setup

### CODE REVIEW

File: src/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+INFURA_API_KEY=O8UtwLH2uVXXIu89dieJ
```
Comment (GregTheGreek) [inline src/.env]: you put it back....

File: src/components/tokens/ListItem.js [hunk 2]
```diff
@@ -34,6 +34,9 @@ class CoinListItem extends Component {
    */
   renderPress(coin) {
     if (this.state.totalTaps == 0) {
+      console.log("render press");
+      console.log(coin);
+
```
Comment (GregTheGreek) [inline src/components/tokens/ListItem.js]: remove log

File: src/.env (hunk 1 above)
Comment (CFDevelopment) [reply src/.env]: @GregTheGreek fixed

File: src/components/tokens/ListItem.js (hunk 2 above)
Comment (CFDevelopment) [reply src/components/tokens/ListItem.js]: @GregTheGreek fixed

**Pull request merged**